stdlib/public/core/CTypes.swift

```diff
@@ -130,19 +130,6 @@ public struct OpaquePointer : Equatable, Hashable {
     self.init(unwrapped)
   }
 
-  /// Unsafely convert an unmanaged class reference to an opaque
-  /// C pointer.
-  ///
-  /// This operation does not change reference counts.
-  ///
-  ///     let str0: CFString = "boxcar"
-  ///     let bits = OpaquePointer(bitPattern: Unmanaged(withoutRetaining: str0))
-  ///     let str1 = Unmanaged<CFString>(bitPattern: bits).object
-  @_transparent
-  public init<T>(bitPattern bits: Unmanaged<T>) {
```

Comment:
Could you add an unavailable declaration for the benefit of those developers who migrated their code to current swift.org toolchains?